challenges/binary-search/binary_search.py

```diff
@@ -1,18 +1,34 @@
-def binary_search(sorted_array, test_value):
+def sub_binary_search(sorted_array, test_value, low, high):
     """ run through sorted_array and look for test_value.
     if test_value is in sorted_array, return index for test_value
     in sorted_array. If test_value is not in sorted_array, return
     -1 """
-    counter = 0
+    if low > high:
+        False
+    else:
+        mid = (low+high)//2
+        if test_value == sorted_array[mid]:
+            print(mid)
+            return mid
+        elif test_value < sorted_array[mid]:
+            return sub_binary_search(sorted_array, test_value, low, mid-1)
+        else:
+            return sub_binary_search(sorted_array, test_value, mid+1, high)
+
+
+def binary_search1(sorted_array, test_value, low, high):
```

Comment:
Since this is the base of your recursion, the parameters should match those given in the spec. i.e. only the array and value.